03-Security-In-CICD/Readme.md

```diff
@@ -240,6 +242,21 @@ what we have seen in the case of ad hoc SVT runs:
 
 [Back to top...](Readme.md#contents)
 
+
+### FAQs
+#### I have enabled AzSDK_SVTs task in my release pipeline. I am getting an error ‘The specified module 'AzSDK' was not loaded because no valid module file was found in any module directory’. How do I resolve this issue?
+- Go to ‘AzSDK_SVTs’ task in your release definition.
+- Make sure that the check box  ‘Do not auto-update AzSDK’ is unchecked.
```

Comment:
Give little more background on how this switch would help in fixing the issue. It just keeps the user wondering on how did it fix or going to fix?